docs/06.upgradenotes/10-9/page.md

```diff
@@ -5,6 +5,20 @@ title: Upgrade notes for 10.8 -> 10.9
 
 ## General notes
 
+The 10.9 release has a small number of changes that require special consideration for upgrade:
+
+* Objects that are in **Administrator left hand menu**
+
+
+## Preside objects that are in Administrator left hand menu
```

Comment:
Title should really be:

> Admin interfaces that have been built with the "crudadmin" tool

Where they sit in the navigation is not relevant. Suggest that we put this section _after_ "Coldbox 4" which is the biggest consideration.